tests/gis_tests/geoapp/tests.py

```diff
@@ -461,7 +461,11 @@ def test_relate_lookup(self):
             intersects_mask = 'overlapbdyintersect'
 
         # Testing contains relation mask.
-        self.assertEqual('Texas', Country.objects.get(mpoly__relate=(pnt1, contains_mask)).name)
+        if connection.features.supports_transform:
+            self.assertEqual(
+                Country.objects.get(mpoly__relate=(pnt1, contains_mask)).name,
+                'Texas',
+            )
```

Comment:
MariaDB doesn't support the `Transform` function so `geom` must have the same SRID.